README.md

```diff
@@ -1,5 +1,7 @@
 # pyPESTO - Parameter EStimation TOolbox for python
 
+<image src="https://raw.githubusercontent.com/ICB-DCM/pyPESTO/master/doc/logo/logo_wordmark.svg" width="50%" alt="pyPESTO logo"/>
```

Comment:
```suggestion
<img src="https://raw.githubusercontent.com/ICB-DCM/pyPESTO/master/doc/logo/logo_wordmark.svg" width="50%" alt="pyPESTO logo"/>
```

Check https://github.com/ICB-DCM/pyPESTO/tree/feature_logo. that it is shown properly

Reply:
unfortunately, in the feature branch it cannot be shown correctly yet, because it references a file on the master branch (absolute path). I inserted the absolute path here (to raw.githubusercontent), because with relative links the pypi rendering would fail.

so once the changes are on master, it will show correctly. if there's a better way, welcome :)

Reply:
I see. Yeah, it's annoying to get it working locally offline, on github, pypi, etc... okay then